### DISCUSSION TITLE
Solana

### DISCUSSION THREAD

**Discussion by GoldTaro** [Huniko-Team/community#193] [Q&A] [answered]
What are the key components of the Solana ecosystem?

**Comment by Tvenus** [comment, marked as answer]
Solana Cluster: A set of validators working together to maintain the ledger.
Solana Program Library (SPL): A collection of on-chain programs.
Solana Wallets: Tools for managing SOL tokens and interacting with dApps.
Solana DApps: Decentralized applications built on Solana.
